src/main/java/TaskList.java

```diff
@@ -0,0 +1,103 @@
+import java.util.ArrayList;
+
+public class TaskList {
+
+  public ArrayList<Listing> list;
+
+  /**
+   * Takes in a new Arraylist<Listing> and initialises list
```

Comment:
I think you're missing the parameters of the constructor in your docs?

Reply:
Changed it accordingly